### ISSUE TITLE
Running on GPU freezes

### ISSUE THREAD

**Issue by JeffreyXBao** [tensorflow/tfjs#2687]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
6.13.4
#### Browser version
node.js v13.6.0
#### Describe the problem or feature request
Running using tfjs-node finishes very quickly without any noticable freezes.
Running with tfjs-node-gpu freezes on:
"Epoch 1/100"
After about a minute or two, the next line is:
2020-01-20 20:44:36.479151: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0

#### Code to reproduce the bug / link to feature request
// THIS IS A TEST FILE!!!
const tf = require('@tensorflow/tfjs-node-gpu');

// Optional Load the binding:
// Use '@tensorflow/tfjs-node-gpu' if running with GPU.
// require('@tensorflow/tfjs-node');

// Train a simple model:
const model = tf.sequential();
model.add(tf.layers.dense({units: 100, activation: 'relu', inputShape: [10]}));
model.add(tf.layers.dense({units: 1, activation: 'linear'}));
model.compile({optimizer: 'sgd', loss: 'meanSquaredError'});

const xs = tf.randomNormal([100, 10]);
const ys = tf.randomNormal([100, 1]);

model.fit(xs, ys, {
  epochs: 100,
  callbacks: {
    onEpochEnd: (epoch, log) => console.log(`Epoch ${epoch}: loss = ${log.loss}`)
  }
});

If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

GitHub issues for this repository are tracked in the [tfjs union repository](https://github.com/tensorflow/tfjs/issues).

Please file your issue there, following the guidance in [that issue template](https://github.com/tensorflow/tfjs/blob/master/ISSUE_TEMPLATE.md).

**Comment by jaunkst** [comment]
Freezes on the first epoc for 2 mIns or so for me and then runs through all the training. I am seeing this on other model trainings as well. it is allocating 10Gb of my GPU memory with tfjs-node-js and only 1Gb with thensorflow-gpu in python. If I set the epoc and batch to 1 it will start faster. If I then set it back to say 256 batch and 25 epocs  it just runs immediately but still allocates most of the GPU memory.

**Comment by jaunkst** [comment]
@tensorflow/tfjs-node-gpu@1.4.0 seems to start immediately. 
My testing with fashion-mnist-vae also has the same behavior.

**Comment by kangyizhang** [comment]
hi @JeffreyXBao , can you provide the version of the library where you encounter this issue?

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
